### PR TITLE
Dependency updates

### PR DESCRIPTION
**Pull request by ashfurrow**
Updates dependencies to their latest versions, including a final release of RxSwift. See #720 for more info.

### CODE REVIEW

File: Cartfile [hunk 1]
```diff
@@ -1,3 +1,3 @@
 github "Alamofire/Alamofire" ~> 4.0.0
-github "ReactiveCocoa/ReactiveSwift" "1.0.0-alpha.2"
-github "ReactiveX/RxSwift" "3.0.0-rc.1"
+github "ReactiveCocoa/ReactiveSwift" "1.0.0-alpha.3"
+github "ReactiveX/RxSwift" "3.0.0"
```
Comment (AndrewSB) [inline Cartfile]: I think this should either be `~> 3.0.0` or `== 3.0.0`
https://github.com/Carthage/Carthage/blob/master/Documentation/Artifacts.md#version-requirement

Comment (AndrewSB) [inline Cartfile]: It works as is now, but `""` tell carthage to use `git rev-parse`, which the dependency resolver often has issues with (à la https://github.com/Carthage/Carthage/issues/733)

**Comment by AndrewSB** [comment]
CI errd out for some strange reason, attempting a rebuild

**Comment by pedrovereza** [comment]
Attempting another rebuild now that #745 seems to have saved `cocoapods/repo/master`

**Comment by pedrovereza** [comment]
And it got past `pod install`. I'm so happy :tada:

**Pull request merged**